Mindbox.WorkingCalendar/WorkingCalendar.cs

```diff
@@ -86,7 +86,8 @@ public int CountWorkingDaysInPeriod(DateTime startDateTime, DateTime endDateTime
 			var totalElapsedDays = (int) (endDateTimeRounded - startDateTimeRounded).TotalDays;
 			
 			var daysFromNearestMonday = dayOfWeekOffsets[startDateTimeRounded.DayOfWeek] + totalElapsedDays;
-			var weekendDays = (daysFromNearestMonday / 7) * 2 + (daysFromNearestMonday % 7 == 6 ? 1 : 0);
+			var weekendDays = (daysFromNearestMonday / 7) * 2 + (daysFromNearestMonday % 7 == 6 ? 1 : 0) -
+				(startDateTimeRounded.DayOfWeek == DayOfWeek.Sunday ? 1 : 0);
```

Comment:
```suggestion
			var weekendDays = (daysFromNearestMonday / 7) * 2
				+ (daysFromNearestMonday % 7 == 6 ? 1 : 0)
				- (startDateTimeRounded.DayOfWeek == DayOfWeek.Sunday ? 1 : 0);
```

Давай только вот так напишем, чтобы читалось получше